closure/private/defs.bzl

```diff
@@ -42,16 +43,20 @@ JS_HIDE_WARNING_ARGS = [
     "--hide_warnings_for=.soy.js",
     "--hide_warnings_for=external/closure_library/",
     "--hide_warnings_for=external/soyutils_usegoog/",
+    "--hide_warnings_for=external/protobuf_js/",
```

Comment:
Note: When I write a compiler frontend for `closure_js_binary` (similar to JsChecker) we're going to have a better way of dealing with this—via the suppression system. I'm starting to realize now that we're going to have to propagate suppressions to parent rules on a per-file basis.